### ISSUE TITLE
FPPD crash at midnight

### ISSUE THREAD

**Issue by TrekErs** [FalconChristmas/fpp#1159]
FPP 5.3.1
OS 5.3
Raspberry Pi ZeroW

FPPD Crashed

Noticed multiple log entries at milliseconds after 00:00:00 each day regarding playlist Section position 0 is outside of section MainPlaylist after a few days FPPD seems to crash.  I have 3 Pi Zero's running the same version and playlist and all crashed the same day.

[FPP_Logs_20211120.0233.zip](https://github.com/FalconChristmas/fpp/files/7573785/FPP_Logs_20211120.0233.zip)

**Comment by cpinkham** [comment]
I don't see any errors from around midnight in your logs.  I think the Section position 0 error is a red herring possibly because your restartInternetRadio.sh script is telling FPPD to resume playing a playlist when nothing was playing.  What is in your restartInternetRadio.sh script?  We'll also need to see logs from the midnight time frame.  Are you also running an internet radio script then as well?

Please update to 5.4.1 before trying to reproduce in case this issue has already been fixed.

**Comment by TrekErs** [comment]
It looks like the crash didn’t happen until 6:20pm on Nov 19.  All 3 pi zeros crashed within minutes of this time. I had just noticed the time of the previous log entry noting an issue with the playlist section position 0 and went back and noticed that same entry at midnight for several days prior.  I’ll have to look at the log file I uploaded when I’m on my pc, I hope I didn’t accidentally upload the wrong log file since I don’t have any scripts on those pi zeros showing in the file manager. The only FPP that contains the after hours plug-in is my Pi4. Having said all that I did update to 5.4.1 shortly after release and I haven’t see a repeat of this behavior. Thanks for taking the time to look at this.
